### PR TITLE
feat(Styleguide): Add storybook controls addon [WEB-937]

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
- Bumps storybook to latest release candidate for 6 which comes with some nice improvements.
- Adds storybook controls with one example.
- Changes the brand link default link to just link to the relative root. 

### PR Checklist

- [ ] Related to Abstract designs:
- [x] Related to JIRA ticket: WEB-937
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5efe6d23e5f606ff593a32fd--codecademy-styleguide.netlify.app>https://5efe6d23e5f606ff593a32fd--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5efe6d23e5f606ff593a32fd>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1af93aa27f0461f29ff8d9--codecademy-styleguide.netlify.app>https://5f1af93aa27f0461f29ff8d9--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f1af93aa27f0461f29ff8d9>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1b1516e0fedb9c7b36cb46--codecademy-styleguide.netlify.app>https://5f1b1516e0fedb9c7b36cb46--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f1b1516e0fedb9c7b36cb46>Deploy Logs</a></p>

File: packages/styleguide/.storybook/main.js [hunk 1]
```diff
@@ -18,9 +18,10 @@ module.exports = {
 
   addons: [
     '@storybook/addon-essentials',
-    '@storybook/addon-actions',
     '@storybook/addon-a11y',
     '@storybook/addon-docs',
+    '@storybook/addon-controls',
+    '@storybook/addon-actions',
```
Comment (jakemhiller) [inline packages/styleguide/.storybook/main.js:24]: [addon-essentials](https://github.com/storybookjs/storybook/tree/next/addons/essentials)

has a few of these in it already... but we could remove that and be more explicit if we want.

It includes:

- Actions
- Backgrounds
- Controls
- Docs
- Viewport

Comment (codecaaron) [reply packages/styleguide/.storybook/main.js:24]: Gonna make a ticket to switch to this.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.8.0-alpha.575e0a.0<br/>@codecademy/gamut-styles@4.2.0-alpha.575e0a.0<br/>@codecademy/gamut@12.1.0-alpha.575e0a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1f0a029771fe022b24e105--gamut-preview.netlify.app>https://5f1f0a029771fe022b24e105--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f1f0a029771fe022b24e105>Deploy Logs</a></p>

**Pull request merged**